src/sections/2021/section-2.js

```diff
@@ -0,0 +1,176 @@
+      </GridCell>
+      
+      <GridCell>
+        <Quote cite="Executive, successful design system">
+          <p>It has not only outperformed in terms of KPIs, but it has also allowed us to scale more work with fewer people.</p>
+        </Quote>
+      </GridCell>
+      
+      <GridCell>
+        <h3>Design system teams</h3>
+      </GridCell>
+
+      <GridCell>
+        <Figure count="2.2" caption="171 Responses">
+          <PieChart
+            title="Individual contributors usually suggest design systems"
+            dataPoints={[
+              ['An individual contributor(s) suggested starting a design system', 57],
+              ['Leadership suggested starting a design system', 22],
+              ['A third party suggested starting a design system', 3],
+              ['A combination of one or more of the above', 2],
+              ['I am not sure/other', 16]
+            ]}
+          >
+          <p>Where did the <em>idea</em> for your current design system begin?</p>
+          </PieChart>
+        </Figure>
+      </GridCell>
+      
+      <GridCell>
+        <Figure count="2.3" caption="171 Responses | Respondents were asked to select all that apply from a list of 19 items with an option to enter other answers.">
+          <h4>Color system</h4>
+          <p>95%</p>
+          <h4>Typography system</h4>
+          <p>89%</p>
+          <h4>Form components</h4>
+          <p>89%</p>
+          <h4>Navigation components</h4>
+          <p>74%</p>
+          <h4>Usage guidelines</h4>
+          <p>74%</p>
+          <h4>Spacing system</h4>
+          <p>73%</p>
+          <h4>Design files</h4>
+          <p>71%</p>
+          <h4>CSS code</h4>
+          <p>70%</p>
+          <h4>Framework-specific components</h4>
+          <p>69%</p>
+          <h4>Accessibility guidelines</h4>
+          <h4>65%</h4>
+          <h4>Grid system</h4>
+          <p>64%</p>
+          <h4>Layout system</h4>
+          <p>64%</p>
+          <h4>HTML code</h4>
+          <p>61%</p>
+          <h4>Brand guidelines</h4>
+          <p>60%</p>
+          <h4>JavaScript code</h4>
+          <p>57%</p>
+          <h4>Example page templates</h4>
+          <p>50%</p>
+          <h4>Content blocks</h4>
+          <p>46%</p>
+          <h4>Voice & tone guidelines</h4>
+          <p>36%</p>
+          <h4>Animation system</h4>
+          <p>21%</p>
+          <h4>Other (please specify)</h4>
+          <p>	6%</p>
+        </Figure>
+      </GridCell>
+      
+      <GridCell span="2">
+        <SmallCallout>
+          <p>16 of the 19 elements were all contained in over 50% of the in-house respondents’ design systems.</p>
+        </SmallCallout>
+      </GridCell>
+      
+      <GridCell>
+        <Figure count="2.4" caption="374 Responses">
+          <BarChart
+            styleFormat="large"
+            title="Organization Size"
+            keyMap={[
+              `Which disciplines do you currently have on your design system team?`,
```

Comment:
Doc has some red text here. Can we add it?

Reply:
I added in the `<em>`, but they will be printed out as text not html tags for the moment. We need to fix that in a bug card with the component itself.